src/frame/plane.rs

```diff
@@ -707,9 +707,9 @@ pub mod test {
   }
 
   #[test]
-  fn test_plane_pad() {
```

Comment:
It's your intention to get rid of the test_plane_pad?

Reply:
whoops no, I'll restore it